transform/mattermost-analytics/models/marts/product/calls/fct_calls_activity.sql

```diff
@@ -8,4 +8,5 @@
 select
     {{dbt_utils.star(ref('int_calls_active_days_spined'))}}
 from
-    {{ ref('int_calls_active_days_spined') }}
\ No newline at end of file
+    {{ ref('int_calls_active_days_spined') }} ca
+    where exists (select 1 from {{ ref('int_server_hosting_type') }} si where ca.server_id = si.server_id)
```

Comment:
Let's document why these servers are excluded.

Reply:
This is now documented in Confluence.